Default layout change of issue#2

nick-wang opened the pull request:
- Version 0.3.4
  * Remove the duplicate default value in salt state
  * Not to install package by default
  * Remove salt indent and change the order of pillar

nick-wang committed:
Remove the duplicate default value in salt state except for DRBD resource.

Remove the duplicate default value of sync time and required
device/disk name. But leave the default value for DRBD resource
which may be omitted in pillar.

nick-wang committed:
Change not installing package by default.

nick-wang committed:
Remove salt indent and change the order of pillar

nick-wang committed:
Version 0.3.4, change pillar layout and remove duplicate default value.

nick-wang commented:
The `form.yml` tested on SUMA 4.0

nick-wang commented:
I removed all the duplicated `default` value, except some belong to DRBD resource that may not included in pillar file.

arbulu89 commented on drbd/defaults.yaml:
```diff
@@ -1,15 +1,13 @@
-  with_ha: true
+  install_packages: false
+  with_ha: false
+  promotion: "drbd-node1"
```
I wouldn't put a default promotion node here. I think it's a bit strange. And actually, I think It would be better if the execution fails if you have not put the value, as most of the times this value will be different.

nick-wang replied:
Indeed, it make sense. The parameters more likely different per settings.
I would remove this including `res_template` and `resource` section.

arbulu89 commented on drbd/defaults.yaml:
```diff
@@ -1,15 +1,13 @@
+  install_packages: false
+  with_ha: false
+  promotion: "drbd-node1"
+  res_template: "res_single_vol_v9.j2"
```
The same with the template. I wouldn't put a default template name. 
I don't really know how drbd is used normally, but I would say that there is not a "default" way. It's perfect to provide the templates, I like that, but I would prefer that the user must to choose it explicitily.

nick-wang replied:
The template is quite common can could be reused in generic scenario. I would leave as it is but remove the `promotion` and the following `resource`
Thanks for the opinion.

arbulu89 reviewed (approved):
I like the changes @nick-wang !
I think we could merge.
I have some minor suggestions if you want to have a look there.

nick-wang committed:
Remove the promotion and resource in default.yaml.

Must provide the promotion and resource in pillar file,
as most of the times the values will be different.
